python/langsmith/client.py

```diff
@@ -3412,6 +3414,57 @@ def list_examples(
             if limit is not None and i + 1 >= limit:
                 break
 
+    @ls_utils.xor_args(("dataset_name", "dataset_id"))
+    def search_examples(
+        self,
+        query: dict,
```

Comment:
can we call this input? There are many types of queries we might want to support (metadata filtering, actual query if we make this a retriever eventually, etc) so would rather be explicit for few shot for now?

Reply:
How about make it positional only?

Reply:
oh it is lol

Reply:
i like

Reply:
updated